.travis.yml

```diff
@@ -16,8 +16,15 @@ script:
 - npm run travis:verify
 after_script:
 - "./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT"
+after_success:
+curl -sSL https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master/src/bootstrap.sh | bash -s
 deploy:
   provider: script
   script: bash .travis/deploy.sh
   'on':
     branch: master
+env:
+    global:
+    - REPO="git@github.com:RedHatInsights/insights-service-portal-build"
```

Comment:
I don't like having this in the repo, but I'm assuming this is needed by the downloaded script?  I'd prefer to keep this as a travis-side encrypted env var, and not baked into the file.

Also (and this may not be for this PR) *most* of this information can be derived in a script like a I did here: https://github.com/ManageIQ/service_portal-ui/blob/master/.travis/release.sh#L3-L4 (note that DEPLOY_REPO is a build time env var, so I can change where to point to and not bake it into the .travis.yml)

@iphands ... can we chat about this?